api/app/signals/apps/api/views/signals/public/signals.py

```diff
@@ -69,13 +70,20 @@ def retrieve(self, request, uuid):
         return Response(data)
 
     @action(detail=False, url_path=r'geography/?$', methods=['GET'], filterset_class=PublicSignalGeographyFilter,
-            ordering=('-created_at', ), ordering_fields=('created_at', ))
-    def geography(self, request):
+            ordering=('-created_at', ), ordering_fields=('created_at', ),
+            serializer_class=PublicSignalGeographtSerializer)
+    def geography(self, request) -> Response:
         """
         Returns a GeoJSON of all Signal's that are in an "Open" state and in a publicly available category.
         Additional filtering can be done by adding query parameters.
         """
+        serializer = self.serializer_class(data=request.GET)
+
+        if not serializer.is_valid():
```

Comment:
Wondering what will happen when you just do `serialize.is_valid(raise_exception=True)` here